### ISSUE TITLE
One idea.

### ISSUE THREAD

**Issue by Camilo193** [benkeen/generatedata#319]
Another idea:
One option when you want to generate the code in the same order of two lists.
![generate](https://cloud.githubusercontent.com/assets/14241227/10984017/7ac7e862-83e4-11e5-92ea-9101e026349c.png)

**Comment by benkeen** [comment]
Yeah I've thought about this too! I've been hesitant to add it because it would require a new section added to the page where you can manage your lists. It's definitely a good idea, but I'm a little fearful of bloat: the script used to be exceedingly simple, now I'm finding it increasingly intimidating when you arrive at the Generate page.

But good idea! I'll flag this as a feature request.
